src/graphql/mutations/user.rs

```diff
@@ -50,6 +52,11 @@ impl UserMutation {
             &TokenPayload {
                 user_id: user.id,
                 created: Utc::now(),
+                scopes: scopes
+                    .unwrap_or_else(|| vec!["*".into()])
```

Comment:
this is dangerous!! if it can't find scopes return an empty vec!!

Reply:
I want to delay this until we update the frontend auth system to specify the scopes explicitly (since if this is deployed to production it will break the frontend)

Reply:
I'll update this to use scopes, but it will temporarily break frontend until we update frontend this new API